test/pyTest/testTranscoderTranscodeAudioMov.py

```diff
@@ -52,7 +52,7 @@ def testTranscodeMovVariableNbSamplesPerFrame():
 def testTranscodeMovExtractChannels():
     """
     Transcode the audio stream of a MOV file which contains a video stream.
-    Extract channel one and third of the audio stream (5.1).
+    Extract channel one and third of the audio stream (5.1), and create two output streams.
```

Comment:
According to 
```c++
transcoder.addStream( av.InputStreamDesc(inputFileName, audioStreamIndex, 0) )
transcoder.addStream( av.InputStreamDesc(inputFileName, audioStreamIndex, 3) )
```
it seems it should be: "Extract channel one and __four__ of the audio stream (5.1)..."